lib/web3/formatters.js

```diff
@@ -192,6 +195,9 @@ var outputPostFormatter = function(post){
     }
 
     // format the following options
+    if (post.topics == undefined) {
+        post.topics = []
+    }
     post.topics = post.topics.map(function(topic){
```

Comment:
Can reformat thiss please to:

```js
    if (!post.topics) {
        post.topics = [];
    }
```

(Missing `;` and using `!` is better as the node could return us also `topics = null`)